CHANGELOG.md

```diff
@@ -0,0 +1,15 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+## readit v0.1 - 2018-02-11
+- Initial release supports Python 2.7 and Python 3.6
+### Added
+- Bookmark multiple URLs at a time
+- Bookmark URL with respective Tags
+- Search and display Bookmarks by TAG
+- Display all Bookmarks in table format
+- Remove a Bookmarked URL
```

Comment:
It should be **Remove bookmark using specific URL_ID.**

Reply:
@shitalmule04 
yes. i will make this change.

Reply:
@shitalmule04 @ganeshhubale I don't think it's necessary to specify this much of details in the feature list,  just keeping as it was is enough.